File: backend/src/main/java/org/jdk/project/dto/user/UpdateUserRequest.java
package org.jdk.project.dto.user;

import jakarta.validation.constraints.NotNull;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UpdateUserRequest {
  @NotNull private Long id;
  private String nickname;
  private String email;
  private String phone;
  private String sex;
  private String avatarUrl;
  private Boolean enable;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/user/UpdateUserRequest.kt
package org.jdk.project.dto.user

import jakarta.validation.constraints.NotNull

data class UpdateUserRequest(
    @field:NotNull
    var id: Long? = null,
    var nickname: String? = null,
    var password: String? = null,
    var email: String? = null,
    var phone: String? = null,
    var sex: String? = null,
    var avatarUrl: String? = null,
    var enable: Boolean? = null,
    var remark: String? = null,
)
